bin/crew

```diff
@@ -681,7 +723,7 @@ def compress_doc (dir)
 
   if Dir.exist? dir
     system "find #{dir} -type f ! -perm -200 | xargs -r chmod u+w"
-    system "compressdoc --gzip -9#{@verbose} #{dir}"
+    system "compressdoc --gzip -9 -#{@verbose} #{dir}"
```

Comment:
I don't think we want this change.  `#{@verbose}` could be an empty string.

Reply:
Good point.